### DISCUSSION TITLE
AI機器有專利權嗎

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#300] [General]
美國一名法官布林克馬裁決，法律僅允許「個人」擁有專利，人類才屬於「個人」，機器不算，因此人工智慧（AI）不能因其創作而獲得專利，這樣的權利「只保留給人類」。
在人工智慧發展之餘，倫理道德與法律規範等等問題也都逐漸浮出水面，由於界線的模糊，不同人對AI創作是否擁有專利權而產生不同的看法，是個具有爭議的問題。
[https://www.bnext.com.tw/article/64893/machine-patent-allow-in-us](https://www.bnext.com.tw/article/64893/machine-patent-allow-in-us)

**Comment by chun1203** [comment]
很有趣的議題，但如果AI擁有專利權，這樣要如何跟AI申請專利呢

**Comment by MAKERRRofKEVIN** [comment]
之前有分享過一篇關於用機器窮舉來產生所有旋律的酷專案就很適合拿來討論這個案例，那個案例同樣是判決機器無法取得專利，或許短期間內我們還很難看到機器被允許取得專利的案例，但這過程中的思辨還是很值得看看

**Comment by 20010906** [comment]
不同國家對於AI是否能擁有專利權的看法不一，我蠻好奇在未來國家之間對於AI的想法以及法規應如何達成共識

**Comment by 109302003** [comment]
Ai的新興衍伸出的法律、道德倫理問題值得我們深思。

**Comment by YuHengChen54** [comment]
AI擁有自己的意識之後真的衍生出很多問題
